<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mobsatstatus</class>
 <widget class="QWidget" name="mobsatstatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytMbstStts">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytMbstStts
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytMbstStts.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmmndBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/cmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnBtn">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>10</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstBtn">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>10</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pwrBtn">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>10</y>
     <width>151</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pwrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pwrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>376</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clrBtn">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>376</y>
     <width>153</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/clrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tempClrBtn">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tempClrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/tempClrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckBtn">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clpPrxyPedBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>155</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clpPrxyPedBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rerClpPrxyBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>188</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#rerClpPrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="frtClpPrxyBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frtClpPrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dirPrxyBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>252</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dirPrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rerMicSwBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>281</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#rerMicSwBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="frtMicSwBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frtMicSwBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedClpBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>186</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedClpBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rerclpBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>217</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#rerclpBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="frtClpBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>248</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frtClpBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dr1MicSwBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>280</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dr1MicSwBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dr2MicSwBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dr2MicSwBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbsMotBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>340</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbsMotBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedParkSttsMbsBtn">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>157</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedParkSttsMbsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedPwrSttsMbsBtn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>339</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedPwrSttsMbsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
